user-docs/modules/ROOT/pages/sop-branching-fedora-iot.adoc

```diff
@@ -0,0 +1,189 @@
+= Branching Fedora IoT
+
+Branched is the name given to a version of Fedora that has "branched" from the rolling Rawhide tree and will become the next stable Fedora release. This document will detail the steps for branching Fedora IoT from Rawhide. 
+
+To complete these steps and open a pull request you should only need a valid [Fedora account](https://docs.fedoraproject.org/en-US/fedora-accounts/user/).
+
+== Working with the pungi-iot repository
+
+Clone and fork the Fedora IoT Pungi repository to make changes to the configuration files used to build Fedora IoT.
+
+----
+git clone https://pagure.io/fedora-iot/pungi-iot.git
+----
+
+
+Create the new branch and push to the upstream repository. Change `$release` to the new numeric branch of Fedora.
+This is not done as a PR and should be pushed directly to the upstream repository. 
+
+----
+git checkout main; git pull; git checkout -b f$release; git push --set-upstream upstream f$release
+----
+
+Example, used for Fedora 40:
+
+----
+git checkout main; git pull; git checkout -b f40 ; git push --set-upstream upstream f40
```

Comment:
It should be noted that creating a new branch on origin currently can not be done by a PR so repo access is needed here.

Reply:
Mentioned above this, added a note about permissions.